sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java

```diff
@@ -21,7 +22,24 @@
  */
 public class SensitivityAnalysisParameters extends AbstractExtendable<SensitivityAnalysisParameters> {
 
-    public static final String VERSION = "1.0";
+    public static final String VERSION = "1.1";
+
+    static final double FLOW_SENSITIVITY_VALUE_THRESHOLD_DEFAULT_VALUE = 0.0;
+    static final double VOLTAGE_SENSITIVITY_VALUE_THRESHOLD_DEFAULT_VALUE = 0.0;
+    static final double FLOW_VOLTAGE_SENSITIVITY_VALUE_THRESHOLD_DEFAULT_VALUE = 0.0;
+    static final double ANGLE_FLOW_SENSITIVITY_VALUE_THRESHOLD_DEFAULT_VALUE = 0.0;
+
+    @JsonProperty("flow-sensitivity-value-threshold")
```

Comment:
Do we need this as we have a custom serializer/deserializer?

Reply:
No I think it is useless.